src/hermes/commands/deposit/base.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: 2023 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-FileContributor: David Pape
+
+import click
+
+from hermes.model.context import CodeMetaContext
+
+
+class BaseDepositPlugin:
```

Comment:
Are there reasons why you didn't make this an ABC?

Reply:
ABCs force the inherited class to implement all methods. E.g. in the `FileDepositPlugin`, this would mean 6 or so additional methods that all `pass` because there's nothing meaningful for them to do.

Reply:
**opinion:** `ABC` are unpythonic